scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A script to check that travis.yml file & protractor.conf.js have the
+same test suites."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import re
+
+import python_utils
+import utils
+
+
+def extract_travis_jobs_section():
+    """Extracts the env/jobs section from the .travis.yml file.
+
+    Returns:
+        list(str). A list of all jobs in travis.yml file.
+    """
+    travis_file = read_travis_yml_file()
+    jobs_raw = travis_file['env']['jobs']
+    suites_from_jobs = []
+    for job in jobs_raw:
+        # The jobs are in the form RUN_E2E_TESTS_ACCESSIBILITY=true
+        # so the following line removes the RUN_E2E_TESTS_(14 length)
+        # and =true(5 length) part.
+        suites_from_jobs.append(utils.snake_case_to_camel_case(job[14:-5].lower()))
+    suites_from_jobs.sort()
+
+    return suites_from_jobs
+
+
+def extract_travis_script_section():
+    """Extracts the script section from the .travis.yml file.
+
+    Returns:
+        list(str). A list of all scripts in travis.yml file.
+    """
+    travis_file = read_travis_yml_file()
+    script_raw = travis_file['script']
+    # The following line extracts the test suites from patterns like
+    # --suite="accessibility" --
+    hyphen_between_regex = re.compile(r'--suite="(.+?)" --')
+    suites_from_script = []
+
+    for script in script_raw:
+        matches = hyphen_between_regex.finditer(script)
+        for match in matches:
+            # The extracted data is of the form --suite="subscriptions" --
+            # hence 9 to remove --suite=" and -4 to remove " --
+            suites_from_script.append(match.group()[9:-4])
```

Comment:
Here and elsewhere, use regex and capture groups (see above).

Reply:
Done